transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -208,40 +208,40 @@ public static int sendFd(int socketFd, int fd) throws IOException {
-    public static native int getTcpUserTimeout(int milliseconds);
-    public static native int isIpFreeBind(int fd);
-
-    public static native void setReuseAddress(int fd, int reuseAddress);
-    public static native void setReusePort(int fd, int reuseAddress);
-    public static native void setTcpFastopen(int fd, int tcpFastopenBacklog);
-    public static native void setTcpNotSentLowAt(int fd, int tcpNotSentLowAt);
-    public static native void setTrafficClass(int fd, int tcpNoDelay);
-    public static native void setBroadcast(int fd, int broadcast);
-    public static native void setTcpKeepIdle(int fd, int seconds);
-    public static native void setTcpKeepIntvl(int fd, int seconds);
-    public static native void setTcpKeepCnt(int fd, int probes);
-    public static native void setTcpUserTimeout(int fd, int milliseconds);
-    public static native void setIpFreeBind(int fd, int freeBind);
-    public static void tcpInfo(int fd, EpollTcpInfo info) {
+    public static native int isReuseAddress(int fd) throws IOException;
+    public static native int isReusePort(int fd) throws IOException;
+    public static native int getTcpNotSentLowAt(int fd) throws IOException;
+    public static native int getTrafficClass(int fd) throws IOException;
+    public static native int isBroadcast(int fd) throws IOException;
+    public static native int getTcpKeepIdle(int fd) throws IOException;
+    public static native int getTcpKeepIntvl(int fd) throws IOException;
+    public static native int getTcpKeepCnt(int fd) throws IOException;
+    public static native int getTcpUserTimeout(int milliseconds) throws IOException;
+    public static native int isIpFreeBind(int fd)throws IOException;
+
+    public static native void setReuseAddress(int fd, int reuseAddress) throws IOException;
+    public static native void setReusePort(int fd, int reuseAddress) throws IOException;
+    public static native void setTcpFastopen(int fd, int tcpFastopenBacklog) throws IOException;
+    public static native void setTcpNotSentLowAt(int fd, int tcpNotSentLowAt) throws IOException;
+    public static native void setTrafficClass(int fd, int tcpNoDelay) throws IOException;
+    public static native void setBroadcast(int fd, int broadcast) throws IOException;
+    public static native void setTcpKeepIdle(int fd, int seconds) throws IOException;
+    public static native void setTcpKeepIntvl(int fd, int seconds) throws IOException;
+    public static native void setTcpKeepCnt(int fd, int probes) throws IOException;
+    public static native void setTcpUserTimeout(int fd, int milliseconds)throws IOException;
+    public static native void setIpFreeBind(int fd, int freeBind) throws IOException;
+    public static void tcpInfo(int fd, EpollTcpInfo info) throws IOException {
         tcpInfo0(fd, info.info);
     }
 
-    private static native void tcpInfo0(int fd, int[] array);
+    private static native void tcpInfo0(int fd, int[] array) throws IOException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)